omegatrade-app/backend/app/controller/dashboard.controller.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+const Company = require('../models/company.model')
+
+/**
+ * Function to get stockdata of a company
+ * @param {\} req 
+ * @param {*} res 
+ */
+exports.getStockData = async function (req, res) {
+    try {
+        const companyId = req.params.companyId || null
+        if (companyId) {
+            const date = req.query.date ? parseFloat(req.query.date) : null;
+            const [ company ] = await Company.getCompanySimulation(companyId);
+            const stocks = await Company.getStocks(companyId, date);
+            res.status(200).json({
+                success: true,
+                data: {
+                    company: company,
+                    stocks: stocks
+                }
+            });
```

Comment:
Missing return

Reply:
the above suggestion has been added and handled error in controllers.